### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by GabrielCiolac**

### CODE REVIEW

**Comment by Mathew0876** [comment]
Looks nice

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
-# SYSC4806 [![Build Status](https://travis-ci.com/GabrielCiolac/SYSC4806.svg?branch=main)](https://travis-ci.com/GabrielCiolac/SYSC4806)
\ No newline at end of file
+# SYSC4806 [![Build Status](https://travis-ci.com/GabrielCiolac/SYSC4806.svg?branch=development)](https://travis-ci.com/GabrielCiolac/SYSC4806)
```
Comment (Mathew0876) [inline README.md:1]: main?

**Review by sammytripp — CHANGES_REQUESTED** [review]
Change branch name

**Review by thatSoftwareGirl — APPROVED** [review]
Nice

**Review by Mathew0876 — APPROVED** [review]
lit

**Review by sammytripp — APPROVED** [review]

**Pull request merged**